.gitignore

```diff
@@ -1,3 +1,6 @@
+# Environment variables
+.envrc
```

Comment:
Can you add the instructions about env vars and using direnv to the `README`? You can prob copy from another backend's README